### PR TITLE
docs(recipes): update automock recipe for v2.0

### PR DESCRIPTION
**Pull request by omermorad**
## PR Checklist
- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## Description
As Automock releases version 2.0, it is recommended that the Automock recipe documentation be updated.
 - Since Automock now allows additional adapters, the '@automock/adapters.nestjs' package must be installed.
 - Provide more illustrative examples
 - Fix repository URL and link to online documentation.

### CODE REVIEW

**Review by jmcdo29 — APPROVED** [review]
The updates look nice, the example is very clear. So long as Kamil likes it, I'd say it's good to go!

**Comment by omermorad** [comment]
Thanks 🚀

Kamil, I'd appreciate it if we could wait one or two days so I can publish the packages and release notes 🙏🏻

File: content/recipes/automock.md [hunk 1]
```diff
@@ -1,172 +1,229 @@
...
+is automated, simplifying the unit test setup process.
 
 #### Installation
 
+Automock support both Jest and Sinon. Just install the appropriate package for your testing framework of choice.
+Furthermore, you need to install the `@automock/adapters.nestjs` (as Automock supports other adapters).
+
 ```bash
-$ npm i -D @automock/jest
+npm i -D @automock/jest @automock/adapters.nestjs
```
Comment (Tony133) [inline content/recipes/automock.md:28]:
```suggestion
$ npm i -D @automock/jest @automock/adapters.nestjs
```

File: content/recipes/automock.md [hunk 2]
```diff
@@ -1,172 +1,229 @@
...
 ```
 
-Automock does not require any additional setup.
+Or, for Sinon:
 
-> info **info** Jest is the only test framework currently supported by Automock.
-Sinon will shortly be released.
+```bash
+npm i -D @automock/sinon @automock/adapters.nestjs
```
Comment (Tony133) [inline content/recipes/automock.md:34]:
```suggestion
$ npm i -D @automock/sinon @automock/adapters.nestjs
```

**Review by Tony133 — COMMENTED** [review]
just to be in line with the rest of the NestJS documentation 🙂, otherwise I would say it's fine.😉👍

**Review by Tony133 — APPROVED** [review]

**Comment by omermorad** [comment]
@jmcdo29 With the release of v2.0, I'm almost there.

While revisiting the NestJS documentation, I had an idea. Unit testing is covered in the "Fundamentals > Testing" section (https://docs.nestjs.com/fundamentals/testing#unit-testing). Perhaps introducing Automock as an optional testing strategy (under "Auto Mocking" or "Utilities") makes sense?

Also, this is also an opportunity to introduce the new documentation website, which can be found at https://automock.dev/docs/ and https://automock.dev/docs/adapters/intro.

CC: @kamilmysliwiec

Thanks!

**Review by micalevisk — APPROVED** [review]

**Pull request merged**